src/_data/navigation.json

```diff
@@ -20,5 +20,6 @@
       "text": "Playground",
       "url": "/play"
     }
-  ]
+  ],
+  "donate": "/donate"
```

Comment:
If we are going to add this here, we should probably have both the URL and the text like we do for the other links.

Reply:
Can we change it like this?
```
{
"items" : [...]
"donate" : { text: "donate", URL: "/donate" }
}

```
HTML: 
```
<a href="{{ navigation.donate.url }}" class="c-btn c-btn--primary donate-link">{{navigation.donate.text}}</a>
```

Reply:
Yup, that will work.